src/deploy/osp_deployer/director.py

```diff
@@ -932,35 +934,34 @@ def setup_unity_manila(self, unity_manila_yaml):
 
         overcloud_images_file = self.home_dir + "/overcloud_images.yaml"
 
-        cmds = [
-            'sed -i "s|<manila_unity_driver_handles_share_servers>|' +
-            self.settings.manila_unity_driver_handles_share_servers +
-            '|" ' + unity_manila_yaml,
-            'sed -i "s|<manila_unity_nas_login>|' +
-            self.settings.manila_unity_nas_login + '|" ' +
-            unity_manila_yaml,
-            'sed -i "s|<manila_unity_nas_password>|' +
-            self.settings.manila_unity_nas_password + '|" ' +
-            unity_manila_yaml,
-            'sed -i "s|<manila_unity_nas_server>|' +
-            self.settings.manila_unity_nas_server + '|" ' +
-            unity_manila_yaml,
-            'sed -i "s|<manila_unity_server_meta_pool>|' +
-            self.settings.manila_unity_server_meta_pool + '|" ' +
-            unity_manila_yaml,
-            'sed -i "s|<manila_unity_share_data_pools>|' +
-            self.settings.manila_unity_share_data_pools + '|" ' +
-            unity_manila_yaml,
-            'sed -i "s|<manila_unity_ethernet_ports>|' +
-            self.settings.manila_unity_ethernet_ports + '|" ' +
-            unity_manila_yaml,
-            'sed -i "s|<manila_unity_ssl_cert_verify>|' +
-            self.settings.manila_unity_ssl_cert_verify + '|" ' +
-            unity_manila_yaml,
-            'sed -i "s|<manila_unity_ssl_cert_path>|' +
-            self.settings.manila_unity_ssl_cert_path + '|" ' +
-            unity_manila_yaml,
-        ]
+        cmds = ['sed -i "s|<manila_unity_driver_handles_share_servers>|' +
```

Comment:
Changes like this make it difficult and time consuming to review a patch.  It doesn't look like there were any changes made other than code reformatting, but it's difficult to tell.  Please create a separate patch for changes that only involve code formatting changes.

Reply:
Yes you are right will take care of this in the future